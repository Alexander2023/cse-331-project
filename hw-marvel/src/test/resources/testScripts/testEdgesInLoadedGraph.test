# Tests that the edges in the csv file are added to the graph

LoadGraph small small.csv

ListChildren small c1
ListChildren small c2
ListChildren small c3