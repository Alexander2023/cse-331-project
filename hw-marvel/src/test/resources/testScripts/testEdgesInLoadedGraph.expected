# Tests that the edges in the csv file are added to the graph

loaded graph small

the children of c1 in small are: c2(b1)
the children of c2 in small are: c1(b1)
the children of c3 in small are: